.github/workflows/auto-updates.yaml

```diff
@@ -191,7 +197,7 @@ jobs:
         # If we don't run this before the "git diff-index" it seems to list
         # every file that's been touched by codegen.
         git status
-        echo "create_pr=false" >> $GITHUB_ENV
+        echo "create_pr=${{ needs.meta.outputs.pr_empty_deps }}" >> $GITHUB_ENV
```

Comment:
This seems like we now create a PR if go.mod, go.sum or modules.txt has changed? All other cases are already covered? Just curious (I'm fine with this change I think) if there's a reason why we use continue in them in 201-208 and if we can just remove those special cases and not introduce a new flag? I just don't understand enough of this tbh.

Reply:
I think it was useless churn on the day to day PRs but we want it for releases.

Reply:
In the normal case, a version bump in `go.mod` or the other module accounting files isn't worth carrying over to the other repos unless it changes something in `vendor/`.

For the "Releasability" action the downstream repos, we check that the `go.mod` entries for serving, eventing, etc points to the released version of serving/eventing (e.g. `v0.22.0`, not to one of the made-up "versions" that correspond to a particular commit (e.g. `v0.21.1-0.20210330142021-dca8b0acf039`). If none of the vendored files have changed, this action won't create the PRs needed to move the `main` branches into the "Releaseable" state, and we'll have more by-hand work to do.